tests/ssp/enc/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+//# Copyright (c) Open Enclave SDK contributors.
+//# Licensed under the MIT License.
+
+oeedl_file(../ssp.edl enclave gen)
+
+add_enclave(TARGET ssp_enc UUID f39f25d8-266d-47ad-b9d3-77fb62100d14 SOURCES enc.cpp ${gen})
+
+set_source_files_properties(
+  enc.cpp
+  PROPERTIES
+  COMPILE_FLAGS -fstack-protector-all)
```

Comment:
Is this flag actually file specific or should it be `target_compile_options(ssp_enc)`?

Reply:
The global enclave compiling option is capable of turn on ssp option -fstack-protector-strong. But here for this test we want to make the enclave failed but the host side can detect the failure and then report result is OK rather than segment fault. There may be some elegant solution but this way is easy to make the test pass.